.github/workflows/publish-ghcr.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish Clementine on GHCR
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'develop'
+    tags:
+      - 'v*'
+    paths: 
+      - "src/**"
+      - "Cargo.toml"
+      - "Cargo.lock"
+  pull_request:
+    branches:
+      - 'main'
+    paths: 
+      - "src/**"
+      - "Cargo.toml"
+      - "Cargo.lock"
+  workflow_dispatch:
```

Comment:
Probably is enough on a `tags`

Reply:
I already tested the possibility to push into GHCR: it worked.